.circleci/config.base.yml

```diff
@@ -17,10 +17,7 @@ node12: &node12
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: ${AWS_ECR_ACCOUNT_URL}/amplify-cli-e2e-base-image-repo:latest
-      aws_auth:
-        aws_access_key_id: $ECR_ACCESS_KEY
-        aws_secret_access_key: $ECR_SECRET_ACCESS_KEY
+    - image: ${AWS_ECR_ACCOUNT_URL}/amplify-cli-e2e-base-image-repo-public:latest
```

Comment:
We use different ENV variable to keep the credentials separate and have minimal permission for the ECR credentials. With the removal of this would it be possible to still keep them sepearate?

Reply:
the ERC creds are still separate. unfortunately the circleci aws_auth config does not support session_token and cannot be forked. So the only way to do it is to make the repo public so that no auth is required upon pull.

Reply:
So the options we have are either
 - Use a long lived credentials which only allows pulling the image
 - Make the pull for repo public
I would rather make the repo public in that case and not be bothered with it. But for writing we will have to still use credentials

Reply:
Yes - I've done that in this pull request: https://github.com/aws-amplify/amplify-cli/pull/7736